nrf_802154_sl/include/fem/nrf_fem_protocol_api.h

```diff
@@ -0,0 +1,235 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA. All Rights Reserved.
+ *
+ * The information contained herein is confidential property of Nordic Semiconductor ASA.
+ * The use, copying, transfer or disclosure of such information is prohibited except by
+ * express written agreement with Nordic Semiconductor ASA.
+ */
+
+/**
+ * @brief Protocol interface for Power Amplifier (PA) and Low Noise Amplifier (LNA).
+ *
+ * This module enables toggling of GPIO pins before and after the radio transmission and the radio reception
+ * in order to control a Power Amplifier or a Low Noise Amplifier, or both.
+ *
+ * The application must first provide PA and LNA device-specific configuration parameters to this module.
+ * The protocol must then provide PA and LNA protocol configuration parameters before it can use the functionality.
+ *
+ * When the PA/LNA module is configured, the stack can call the provided enable functions before radio activity
+ * to enable the PA or LNA timer configurations for the upcoming radio activity.
+ * By default, PA/LNA is automatically deactivated on the radio DISABLED event.
+ * This can be disabled, so that a manual deactivation can be performed instead.
+ */
+
+#ifndef NRF_FEM_PROTOCOL_API_H__
+#define NRF_FEM_PROTOCOL_API_H__
+
+#include "nrf_fem_control_config.h"
+
+#include "drivers/nrfx_errors.h"
```

Comment:
This and below should use angle brackets.

Reply:
It would be against Nordic Firmware Coding Standard rule 4.5.1. Is any conflicting coding standard used in this repository?

Reply:
No, since it's an external project, you can follow Nordic standard, I'm just surprised it's defined like this in there, since CPP docs make it clear, that "" should be rather used for headers located in relative paths to the source file:
https://gcc.gnu.org/onlinedocs/cpp/Search-Path.html
Zephyr follows this convention, but you don't need to comply here. Just ignore those comments.